pyoffers/models/application.py

```diff
@@ -0,0 +1,25 @@
+from pyoffers.utils import Filter
+
+from .core import Model, ModelManager
+
+
+class Application(Model):
+    pass
+
+
+class ApplicationManager(ModelManager):
+    name = 'application'
+    model = Application
+
+    def find_all_offer_categories(self, sort=(), fields=None, **kwargs):
+        assert fields is None or isinstance(fields, (tuple, list)), 'Fields should be a tuple or list'
+        return self._call(
+            'findAllOfferCategories',
+            filters=Filter(**kwargs),
+            sort=sort,
+            fields=fields, single_result=False
+        )
+
+    def find_all_offer_category_offer_ids(self, id):
+        result = self._call('findAllOfferCategoryOfferIds', id=id, single_result=False, raw=True)
+        return [int(i) for i in result]  # HO returns ID's as strings, cast to int for convenience
```

Comment:
Please, remove casting. It makes the client behavior inconsistent. In some places it is casted, in some - not. Let's not introduce extra behavior on the top of HO.

Reply:
done